core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -123,6 +123,10 @@ abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
     */
   def listBillingAccountsUsingServiceCredential(implicit executionContext: ExecutionContext): Future[Seq[RawlsBillingAccount]]
 
+  def getProjectBillingAccount(projectName: RawlsBillingProjectName, userInfo: UserInfo)(implicit executionContext: ExecutionContext): Future[Option[String]]
+
+  def getParentBillingAccount(billingProjectId: String, userInfo: UserInfo)(implicit executionContext: ExecutionContext): Future[Option[String]]
```

Comment:
In the spirit of precise naming as stated above:

```suggestion
  def getBillingAccountForGoogleProject(googleProjectId: String, userInfo: UserInfo)(implicit executionContext: ExecutionContext): Future[Option[String]]
```